zmsapi/src/Zmsapi/ClusterQueue.php

```
@@ -23,7 +23,7 @@ public function readResponse(
         \Psr\Http\Message\ResponseInterface $response,
         array $args
     ) {
-        (new Helper\User($request))->checkRights('basic');
+        (new Helper\User($request))->checkAnyPermission('appointment')
         $query = new Query();
```

PHPStan (2.1.54)

[error] Syntax error, unexpected T_VARIABLE on line 27


(phpstan.parse)